avm/res/kubernetes/connected-clusters/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,31 @@
+targetScope = 'subscription'
+
+metadata name = 'Deploy connected clusters in default configuration'
+metadata description = 'This test deploys a connected clusters.'
```

For consistency with other modules
```suggestion
metadata name = 'Using only defaults'
metadata description = 'This instance deploys the module with the minimum set of required parameters.'
```